interfaces/builtin/snapd_control.go

```diff
@@ -35,22 +41,42 @@ const snapdControlBaseDeclarationSlots = `
     deny-auto-connection: true
 `
 
-// http://bazaar.launchpad.net/~ubuntu-security/ubuntu-core-security/trunk/view/head:/data/apparmor/policygroups/ubuntu-core/16.04/snapd-control
 const snapdControlConnectedPlugAppArmor = `
 # Description: Can manage snaps via snapd.
 
 /run/snapd.socket rw,
 `
 
+type snapControlInterface struct {
+	commonInterface
+}
+
+func (iface *snapControlInterface) SanitizePlug(plug *interfaces.Plug) error {
+	if err := iface.commonInterface.SanitizePlug(plug); err != nil {
```

Comment:
Aha, for future-proofing, I assume. In such case can you comment in the common code so that things are clear. Alternatively just don't do this :)

Reply:
I removed it. I'm a strong believer in YAGNI.